<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DC1 Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_top">
        <item>
         <widget class="PlotWidget" name="arrayMap" native="true"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="traceplot_verticalLayout">
          <item>
           <widget class="PlotWidget" name="traceplot_1" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="traceplot_2" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="traceplot_3" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="traceplot_4" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_bottom">
        <item>
         <widget class="PlotWidget" name="spikeRatePlot" native="true"/>
        </item>
        <item>
         <widget class="PlotWidget" name="miniMapWidget" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_offline_data">
     <property name="title">
      <string>Load offline data...</string>
     </property>
     <addaction name="action_npz"/>
     <addaction name="action_mat"/>
     <addaction name="action_csv"/>
    </widget>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as...</string>
     </property>
     <addaction name="action_save_npz"/>
     <addaction name="action_save_mat"/>
     <addaction name="action_save_csv"/>
    </widget>
    <widget class="QMenu" name="menuExport_image">
     <property name="title">
      <string>Export image...</string>
     </property>
     <addaction name="actionEntire_window"/>
     <addaction name="actionWindow_1"/>
     <addaction name="actionWindow_2"/>
     <addaction name="actionWindow_3"/>
     <addaction name="actionWindow_4"/>
    </widget>
    <addaction name="actionNew_session"/>
    <addaction name="menuLoad_offline_data"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="menuSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuExport_image"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionAnalysis_parameters"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuArray_Map">
     <property name="title">
      <string>Array Map</string>
     </property>
     <addaction name="actionParameter_1"/>
     <addaction name="actionParameter_2"/>
     <addaction name="actionParameter_3"/>
    </widget>
    <widget class="QMenu" name="menuChannel_Traces">
     <property name="title">
      <string>Channel Traces</string>
     </property>
     <addaction name="actionParameter_4"/>
     <addaction name="actionParameter_5"/>
     <addaction name="separator"/>
     <addaction name="actionSpike_threshold"/>
     <addaction name="actionTime_window"/>
    </widget>
    <widget class="QMenu" name="menuSpike_Rate_Plot">
     <property name="title">
      <string>Spike Rate Plot</string>
     </property>
     <addaction name="actionParameter_6"/>
     <addaction name="actionParameter_7"/>
     <addaction name="actionParameter"/>
    </widget>
    <widget class="QMenu" name="menuElectrode_Minimap">
     <property name="title">
      <string>Electrode Minimap</string>
     </property>
     <addaction name="actionParameter_8"/>
     <addaction name="actionParameter_9"/>
     <addaction name="actionParameter_10"/>
    </widget>
    <addaction name="menuArray_Map"/>
    <addaction name="menuChannel_Traces"/>
    <addaction name="menuSpike_Rate_Plot"/>
    <addaction name="menuElectrode_Minimap"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuTop_left">
     <property name="title">
      <string>Top left...</string>
     </property>
     <addaction name="actionArray_map"/>
     <addaction name="actionChannel_Traces_2"/>
     <addaction name="actionSpike_Rate_Plot_2"/>
     <addaction name="actionElectrode_Minimap"/>
     <addaction name="actionTrace_Search"/>
    </widget>
    <widget class="QMenu" name="menuTop_right">
     <property name="title">
      <string>Top right...</string>
     </property>
     <addaction name="actionArray_Map_2"/>
     <addaction name="actionChannel_Traces"/>
     <addaction name="actionSpike_Rate_Plot"/>
     <addaction name="actionElectrode_Minimap_2"/>
     <addaction name="actionTrace_Search_2"/>
    </widget>
    <widget class="QMenu" name="menuBottom_left">
     <property name="title">
      <string>Bottom left...</string>
     </property>
     <addaction name="actionArray_Map_3"/>
     <addaction name="actionChannel_Traces_3"/>
     <addaction name="actionSpike_Rate_Plot_3"/>
     <addaction name="actionElectrode_Minimap_3"/>
     <addaction name="actionTrace_Search_3"/>
    </widget>
    <widget class="QMenu" name="menuBottom_right">
     <property name="title">
      <string>Bottom right...</string>
     </property>
     <addaction name="actionArray_Map_4"/>
     <addaction name="actionChannel_Traces_4"/>
     <addaction name="actionSpike_Rate_Plot_4"/>
     <addaction name="actionElectrode_Minimap_4"/>
     <addaction name="actionTrace_Search_4"/>
    </widget>
    <addaction name="actionChange_layout"/>
    <addaction name="menuTop_left"/>
    <addaction name="menuTop_right"/>
    <addaction name="menuBottom_left"/>
    <addaction name="menuBottom_right"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_light_dark_mode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuFilter_design">
     <property name="title">
      <string>Filter</string>
     </property>
     <addaction name="actionParameter_11"/>
     <addaction name="actionParameter_12"/>
     <addaction name="actionParameter_13"/>
     <addaction name="actionH0_Bandpass"/>
     <addaction name="actionAuto"/>
     <addaction name="actionFast_Bandpass"/>
     <addaction name="actionFaster_Bandpass"/>
     <addaction name="actionLitke"/>
     <addaction name="actionNone"/>
    </widget>
    <addaction name="menuFilter_design"/>
    <addaction name="actionSpike_detection"/>
    <addaction name="actionToggle_profiling"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew_session">
   <property name="text">
    <string>New real-time session...</string>
   </property>
  </action>
  <action name="actionLoad_real_time_stream">
   <property name="text">
    <string>Load real-time stream...</string>
   </property>
  </action>
  <action name="actionChange_layout">
   <property name="text">
    <string>Change layout</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionParameter_1">
   <property name="text">
    <string>Parameter 1</string>
   </property>
  </action>
  <action name="actionParameter_2">
   <property name="text">
    <string>Parameter 2</string>
   </property>
  </action>
  <action name="actionParameter_3">
   <property name="text">
    <string>Parameter 3</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionAnalysis_parameters">
   <property name="text">
    <string>Analysis parameters</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport_to">
   <property name="text">
    <string>Export to...</string>
   </property>
  </action>
  <action name="actionParameter_4">
   <property name="text">
    <string>Toggle highlight spikes</string>
   </property>
  </action>
  <action name="actionParameter_5">
   <property name="text">
    <string>Toggle auto axis</string>
   </property>
  </action>
  <action name="actionParameter_6">
   <property name="text">
    <string>Parameter 1</string>
   </property>
  </action>
  <action name="actionParameter_7">
   <property name="text">
    <string>Parameter 2</string>
   </property>
  </action>
  <action name="actionParameter">
   <property name="text">
    <string>Parameter 3</string>
   </property>
  </action>
  <action name="actionParameter_8">
   <property name="text">
    <string>Parameter 1</string>
   </property>
  </action>
  <action name="actionParameter_9">
   <property name="text">
    <string>Parameter 2</string>
   </property>
  </action>
  <action name="actionParameter_10">
   <property name="text">
    <string>Parameter 3</string>
   </property>
  </action>
  <action name="actionArray_map">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="actionArray_Map_2">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="actionArray_Map_3">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="actionArray_Map_4">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="actionChannel_Traces_2">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="actionChannel_Traces_3">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="actionChannel_Traces_4">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="actionSpike_Rate_Plot_2">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="actionElectrode_Minimap">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="actionParameter_11">
   <property name="text">
    <string>Hierlemann</string>
   </property>
  </action>
  <action name="actionParameter_12">
   <property name="text">
    <string>Modified Hierlemann</string>
   </property>
  </action>
  <action name="actionParameter_13">
   <property name="text">
    <string>Highpass</string>
   </property>
  </action>
  <action name="actionSpike_detection">
   <property name="text">
    <string>Spike detection...</string>
   </property>
  </action>
  <action name="action_npz">
   <property name="text">
    <string>Numpy file (.npz)</string>
   </property>
  </action>
  <action name="action_mat">
   <property name="text">
    <string>MATLAB file (.mat)</string>
   </property>
  </action>
  <action name="action_save_npz">
   <property name="text">
    <string>Numpy file (.npz)</string>
   </property>
  </action>
  <action name="action_save_mat">
   <property name="text">
    <string>MATLAB file (.mat)</string>
   </property>
  </action>
  <action name="actionChannel_Traces">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="actionSpike_Rate_Plot">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="actionElectrode_Minimap_2">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="actionSpike_Rate_Plot_3">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="actionElectrode_Minimap_3">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="actionSpike_Rate_Plot_4">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="actionElectrode_Minimap_4">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="action_save_csv">
   <property name="text">
    <string>CSV file (.csv)</string>
   </property>
  </action>
  <action name="action_csv">
   <property name="text">
    <string>CSV file (.csv)</string>
   </property>
  </action>
  <action name="actionEntire_window">
   <property name="text">
    <string>Entire window</string>
   </property>
  </action>
  <action name="actionWindow_1">
   <property name="text">
    <string>Window 1</string>
   </property>
  </action>
  <action name="actionWindow_2">
   <property name="text">
    <string>Window 2</string>
   </property>
  </action>
  <action name="actionWindow_3">
   <property name="text">
    <string>Window 3</string>
   </property>
  </action>
  <action name="actionWindow_4">
   <property name="text">
    <string>Window 4</string>
   </property>
  </action>
  <action name="actionTrace_Search">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="actionTrace_Search_2">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="actionTrace_Search_3">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="actionTrace_Search_4">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="actionToggle_light_dark_mode">
   <property name="text">
    <string>Toggle light/dark mode</string>
   </property>
  </action>
  <action name="actionToggle_profiling">
   <property name="text">
    <string>Toggle profiling</string>
   </property>
  </action>
  <action name="actionH0_Bandpass">
   <property name="text">
    <string>h0 Bandpass</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionFast_Bandpass">
   <property name="text">
    <string>Fast Bandpass</string>
   </property>
  </action>
  <action name="actionFaster_Bandpass">
   <property name="text">
    <string>Faster Bandpass</string>
   </property>
  </action>
  <action name="actionLitke">
   <property name="text">
    <string>Litke</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSpike_threshold">
   <property name="text">
    <string>Spike threshold...</string>
   </property>
  </action>
  <action name="actionTime_window">
   <property name="text">
    <string>Time window...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
